src/content/docs/pt-br/getting-started.mdx

```diff
@@ -2,122 +2,67 @@
-  <Fragment slot="npm">
-  ```shell
-  # crie um novo projeto com o npm
-  npm create astro@latest
-  ```
-  </Fragment>
-  <Fragment slot="pnpm">
-  ```shell
-  # crie um novo projeto com o pnpm
-  pnpm create astro@latest
-  ```
-  </Fragment>
-  <Fragment slot="yarn">
-  ```shell
-  # crie um novo projeto com o yarn
-  yarn create astro
-  ```
-  </Fragment>
-</PackageManagerTabs>
-
-Nosso [Guia de Instalação](/pt-br/install/auto/) tem instruções detalhadas passo-a-passo de como instalar Astro usando nosso assistente de CLI, de criação de um novo projeto a partir de um repositório Astro existente no GitHub, e de instalação manual.
-
-
-## Aprenda Astro
-
-Veja exemplos de alguns dos conceitos-chave e padrões de um site Astro!
-
-📚 [Adicione sua primeira página](/pt-br/core-concepts/astro-pages/) no seu site.
-
-📚 Leia mais sobre a [estrutura de projetos](/pt-br/core-concepts/project-structure/) do Astro.
-
-📚 Aprenda mais sobre o [roteamento baseado em arquivos](/pt-br/core-concepts/routing/) do Astro.
-
-*... encontre a documentação completa da API na aba **Referência**.*
-
-
-## Estenda Astro
-
-🧰 Comece o seu próximo projeto com um [tema pré-construído](https://astro.build/themes).
-
-🧰 Customize seu site com [plugins e componentes](https://astro.build/integrations/) da comunidade.
-
-🧰 Se inspire visitando nossa [galeria de sites](https://astro.build/showcase).
-
-*... veja nosso [guia em como utilizar integrações](/pt-br/guides/integrations-guide/)*.
-
-
-
-## Junte-se a comunidade
-
-Junte-se a nós no [Discord do Astro](https://astro.build/chat) para compartilhar e conseguir ajuda de uma comunidade ativa e amigável!
-
-💬 Diga oi no canal `#general`!
-
-💬 Pergunte a nossa Equipe de Suporte no canal `#support`!
-
-💬 Compartilhe o que você tem feito no canal `#showcase`!
-
-
-## O que há de novo?
-
-[Blog do Astro](https://astro.build/blog/)
-
-[Histórico de alterações do Astro](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
-
-
-<TranslatorList githubRepo="withastro/docs">
-
-## Traduza
-
-Essa tradução da documentação é trazida até você por todas essas pessoas incríveis. [Nos ajude a traduzir!](https://contribute.docs.astro.build/guides/i18n/)
-</TranslatorList>
-
-## Contribua
-
-Essa documentação é trazida até você por todas essas pessoas colaborativas. [Junte-se a nós no GitHub!](https://github.com/withastro/docs)
-
-<ContributorList githubRepo="withastro/docs" />
+import { CardGrid } from '@astrojs/starlight/components'
+import Card from '~/components/Landing/Card.astro'
+import ListCard from '~/components/Landing/ListCard.astro'
+import SplitCard from '~/components/Landing/SplitCard.astro'
+import Discord from '~/components/Landing/Discord.astro'
+
+<CardGrid>
+  <Card title="O que você vai construir com Astro?" icon="laptop">
+    Explore [temas iniciais do Astro](https://astro.build/themes/) para blogs, portfólios, documentações, páginas iniciais, SaaS, marketing, sites de ecommerce e mais!
```

Comment:
```suggestion
    Explore [temas iniciais do Astro](https://astro.build/themes/) para blogs, portfólios, documentações, páginas iniciais, SaaS, marketing, sites de e-commerce e mais!
```

Reply:
I guess "landing pages" could stay as "landing pages", not _páginas iniciais_. Other name that comes to mind is _página institucional_ but I this would differ from english source material. Other than that LGTM!

Reply:
I used "página inicial" based on a few other translations I've seen before, I guess it's close enough, and also, I kept `ecommerce` as is on the original page, if the spelling is wrong it should be made as a PR to the source material first.